SSA/Projects/LLVMRiscV/Pipeline/urem.lean

```diff
@@ -0,0 +1,44 @@
+import SSA.Projects.LLVMRiscV.PeepholeRefine
+import SSA.Projects.LLVMRiscV.simpproc
+import SSA.Projects.RISCV64.Tactic.SimpRiscV
+
+open LLVMRiscV
+
+
+/-! # UREM  -/
+
+def llvm_urem: Com  LLVMPlusRiscV [.llvm (.bitvec 64), .llvm (.bitvec 64)]
+    .pure (.llvm (.bitvec 64))  := [LV| {
+    ^entry (%x: i64, %y: i64 ):
+      %1 = llvm.urem  %x, %y : i64
+      llvm.return %1 : i64
+  }]
```

Comment:
format nit: can you double-check the spaces? I feel like there should not be a space e.g. between `i64` and `)`, as well as the double space before `:=`